<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TensuraReader</class>
 <widget class="QMainWindow" name="TensuraReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(20, 20, 20);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrainsMono NF</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
	color: #ECECEC;
}</string>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="font">
     <font>
      <family>JetBrainsMono NF</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu {
	color: #ECECEC;
}</string>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_Restart"/>
    <addaction name="action_LoadLink"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="font">
     <font>
      <family>JetBrainsMono NF</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu {
	color: #ECECEC;
}

QMenu:hover {
	color: #141414;
	background-color: rgb(85, 255, 255);
}</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Restart">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Restart</string>
   </property>
   <property name="toolTip">
    <string>Change Reading Site</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About Application</string>
   </property>
  </action>
  <action name="action_LoadLink">
   <property name="text">
    <string>&amp;Load Link</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="tensura.qrc"/>
 </resources>
 <connections/>
</ui>
